docs/source/federation/hosted-gateway-definitions.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Hosted gateway configurations
+description: Managing gateways at scale
```

Comment:
I'm wondering if this set of docs should be more about "pull" vs "push" methods of configuration. Pull being a decent starting point and helpful for local development, but in production a "push" model is a much more robust and safer model. We provide a way to manage that using the CLI that we go over here